data.json

```diff
@@ -601,7 +601,7 @@
         "status": "Arrested",
         "last_known_location": "Unknown",
         "security_organ": "Police",
```

Comment:
If you could also add the time she was taken which should be "13:20 23-07-2024" based on this last tweet of hers.

https://x.com/kebxrungi/status/1815693550102687800

Reply:
Noted